Source/WebInspectorUI/UserInterface/Views/DetailsSection.css

```diff
@@ -180,6 +180,10 @@ body[dir=rtl] .details-section > .header::before {
     border-bottom: 1px solid var(--border-color);
 }
 
+.details-section > .content > .group:has(.row.font-variation) {
```

Comment:
this kinda seems like a layering violation, no? why should `WI.DetailsSection` know about `WI.FontVariationDetailsSectionRow`?